docs/src/content/docs/concepts/amounts.mdx

```diff
@@ -0,0 +1,198 @@
+---
+title: Amounts
+parent: Concepts
+---
+
+import { LinkOut, Tooltip } from '@interledger/docs-design-system'
+
+:::tip[Summary]
+Amounts in Open Payments define monetary values using asset codes, asset scales, and specific amount types. They provide a standardized way to represent money across different currencies and payment scenarios.
+:::
+
+Amounts represent monetary values and are fundamental to all payment operations. Every amount consists of three key components: a numerical value, an asset code, and an asset scale. Understanding these components is crucial for building Open Payments applications.
+
+Amounts are central to every Open Payments operation. Whether you're creating an incoming payment, requesting a quote, or tracking payment progress, all amounts follow the same three-part structure: a value, asset code, and asset scale.
+
+Every amount in Open Payments consists of three components:
```

Comment:
We should mention that wallet addresses are tied to one specific asset, and as a result you can only provide amounts in assets corresponding to the one that a wallet address supports